usmqe_tests/api/grafana/test_gluster_at_a_glance.py

```diff
@@ -0,0 +1,56 @@
+"""
```

Comment:
~~Would it make sense to call this test module `grafana-graphite` (instead of just `grafana`) to match what we already have in https://usmqe-testdoc.readthedocs.io/en/latest/api/grafana-graphite/index.html ? Even though the details differs slightly (we are going to get dashboard descriptions from api instead of the package), the main idea still holds.~~ Ook, the answer is no, this seems to be a different type of test, concerned with dashboard structure only.